server/service/src/invoice_line/stock_in_line/update/generate.rs

```diff
@@ -71,7 +71,7 @@ pub fn generate(
                     on_hold: false,
                     barcode_id: None,
                     overwrite_stock_levels: true,
-                    campaign_id: None,
+                    campaign_id: update_line.campaign_id.clone(),
```

Comment:
`Insert` already has this logic, meaning that when a new Inbound Shipment is created and a Campaign is set for an Item, the stock line is also updated accordingly